app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -88,7 +91,8 @@ class MainActivity : AppCompatActivity() {
             val currentDateFile = File(currentDateDirectoryName)
             val gson = Gson()
 
-            val activityLogFilePath = File("$currentDateFile/$currentDate.json")
+            val activityLogFilePath =
+                File("$currentDateFile/$currentDate.json")
```

Comment:
for properties with long initializers add a line break after '=' sign for better readability

https://kotlinlang.org/docs/reference/coding-conventions.html